Monika After Story/game/script-topics.rpy

```diff
@@ -12905,3 +12905,78 @@ label monika_enjoyingspring:
     else:
         m 2rkc "...but I guess there's no real way to avoid it, is there?"
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_blood_type_stereotype",category=["society"],prompt="Blood stereotypes",random=True))
+
+label monika_blood_type_stereotype:
+    m 1eub "Did you know that some people associate the blood type to personality traits?"
+    m 1eua "Particularly in the japanese culture."
+    m 3eua "It's a belief that started in the 30s, but that was really popularized in the 70s."
+    m 3eub "Even though it has no scientific basis, it's a common superstition amongst asian peoples."
+    m "It's like the astrological signs in Occident."
+    m 1eua "In fact, this belief is so popular that you've probably heard about it."
+    m 2hub "After all, most visual novels tells you what's the blood type of their characters."
+    m 4eub "It's a detail that can help players apprehend the character's personality."
+    m 2lud "But for this game...Even if it has a wiki page, it's creator never specified what blood type the club members were."
+    m 1hub "I was thinking that it could be something fun speculating about, so I tried to guess what blood type we all had."
+    m 2eua "Here's what I found."
+    m 1dsc "Ahem."
+    m 3esb "First, let's take the most obvious one, the AB group."
+    m 3esa "It's a group associated with being creative, calm and composed but also with being very critical and eccentric, and sometimes even two-faced."
+    m 3hksdrb "It's pretty obvious that it would be the blood type of Yuri."
+    m 2ekb "Under the impression of passiveness, she could be very critical when she was feeling confident. You saw it for yourself right?"
+    m 2lud "As for the 'two face' thing..."
+    m 2lssdrb "Let's not talk about that... You know what I'm talking about."
+    if persistent.blood_type == "AB":
```

Comment:
Where are you setting this variable? All these checks fail because `persistent.blood_type` is `None` and it'll always be `None`.

Did you intend for Monika to ask your blood type? If so this should be `persistent._mas_pm_blood_type` and would need a menu to get that type

Reply:
Yes, I intended to write the topic when she ask you about your blood type, but I didn't have the time to do it, so I let those bits of dialogue for when someone want to write the topic.
Also, the problem could be solved if I add:
default persistent.blood_type = None
Am I wrong?

Reply:
Okay, so these will have to be player model (`_mas_pm_`) variables, and this topic would probably be waiting on the other one since it requires it in order to be completed.

Reply:
Persistent vars always default to `None` but we add them anyway to show where they're related to. So you'll need to add that `default` line to the blood type topic.